src/torcontrol.h

```diff
@@ -11,7 +11,7 @@
 #include "scheduler.h"
 
 extern const std::string DEFAULT_TOR_CONTROL;
-static const bool DEFAULT_LISTEN_ONION = true;
+static const std::string DEFAULT_LISTEN_ONION = "0";
```

Comment:
String constant cannot be defined reliably in headers. It must be `extern` and defined in a c++ file. Otherwise it will get included into every single compilation unit that includes this header. Sometimes that runs into linker errors or causes executable bloat at the least.